### DISCUSSION TITLE
Export results

### DISCUSSION THREAD

**Discussion by russellandsusan** [rappen/FetchXMLBuilder#495] [Ideas]
Hi Jonas,

FetchXML Builder is a sensational tool. Truly man. It works wonders!

One question: Have you investigated adding the ability to 'export' the Results view to .csv, .xlsx, or other format?

Thanks. Rus.

**Comment by rappen** [comment]
Sorry for very late response!
Yes the export result would be good - please add an [Issue](https://github.com/rappen/FetchXMLBuilder/issues) what you want 🙂
From the result view, you can today select any records and any columns and then `CTRL+C` and then paste it into Excel.
